test/URLEndpointListenerTest.cc

```diff
@@ -397,4 +440,86 @@ TEST_CASE_METHOD(URLEndpointListenerTest, "Listener with Cert Authentication", "
         CBLTLSIdentity_Release(listenerConfig.tlsIdentity);
 }
 
+#ifdef __APPLE__
+TEST_CASE_METHOD(URLEndpointListenerTest, "Listener with Cert Authentication with External KeyPair", "[URLListener]") {
```

Comment:
My expectation is that this test should fail now as we need an enhancement in LiteCore. There should be a test to test Listener configured with TLSIdentity created with external key callbacks and I expect that that test will pass.

Reply:
Wait, you also do it with `listenerConfig.tlsIdentity = createTLSIdentity(true, withExternalKey);`.